src/main/kotlin/com/github/serverfrog/bitburnerplugin/bitburner/Bitburner.kt

```diff
@@ -0,0 +1,116 @@
+package com.github.serverfrog.bitburnerplugin.bitburner
+
+import com.github.serverfrog.bitburnerplugin.MyBundle
+import com.github.serverfrog.bitburnerplugin.config.BitburnerSettings
+import com.intellij.ide.DataManager
+import com.intellij.notification.NotificationGroup
+import com.intellij.notification.NotificationGroupManager
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.search.FilenameIndex
+import com.intellij.psi.search.GlobalSearchScope
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+import java.util.*
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+class Bitburner {
+    companion object {
+        val extensions = arrayListOf("js", "script")
```

Comment:
val extensions = arrayListOf("js", "script", "ns", "txt")

TODO: Add justification about why we only use js and script extension file.
The api-server.js use the function document.saveFile()
 - https://github.com/danielyxie/bitburner/blob/48f80f25d6fb90edc292481905814c0d5593b5a9/electron/api-server.js#L78
 The function saveFile is declared in Electron.tsx. It is using home.writeToScriptFile()
  - https://github.com/danielyxie/bitburner/blob/f8d977f7f84ef7e5722186a5ec243323ec20abb7/src/Electron.tsx#L98
 The function home.writeToScriptFile() is declared in BaseServer.ts which make two validations one of this
 is the function isScriptFilename().
  - https://github.com/danielyxie/bitburner/blob/f6f023eeb4f9c53452fcb8971f5c4151f369aab6/src/Server/BaseServer.ts#L256
 The function isScriptFilename() is declared in isScriptFilename.ts.
 It only accept two types of files: [`.js`, `.script`]
  - https://github.com/danielyxie/bitburner/blob/f6f023eeb4f9c53452fcb8971f5c4151f369aab6/src/Script/isScriptFilename.ts#L3

Reply:
As this, i think somewhere written down, just a kinda copy of the VS Code Plugin, i took the list out there and put it in here. This Plugin was develop not much after the Steam Publish so many things were unknown or changed over the time

Reply:
Interesting, it makes sense. You did an impressive job 9 months ago.